bool erkennung
uint16 posx_px
uint16 posy_px
uint8 breite_px
uint8 hoehe_px 
